src/gui/resources/ProductList.qml

```diff
@@ -23,13 +23,31 @@ Frame {
             width: parent.width
 
             Text {
+                id: idField
                 Layout.alignment: Qt.AlignLeft
                 Layout.minimumWidth: parent.width * 0.07
 
                 text: "ID: " + model.ID
                 horizontalAlignment: Text.AlignHCenter
                 font.bold: true
                 font.pointSize: 12
+
+                color: inspecctProductlink.containsMouse ? "blue" : "black"
+                font.underline: inspecctProductlink.containsMouse
+
+                MouseArea {
+                    id: inspecctProductlink
+                    anchors.fill: parent
+                    cursorShape: Qt.PointingHandCursor
+                    hoverEnabled: true
+
+                    onClicked: {
+                        productPageLoader.source = ""
```

Comment:
It causes previously loaded element to be unloaded, here it closes product-details window.

Reassignment will also cause previous window to be unloaded,
I separated it into two operations because I found it easier to read that way.